engine/src/main/java/engine/physic/PhysicManager.java

```diff
@@ -47,6 +57,12 @@ public void addPhysicObject(GameObject gameObject) throws AddObjectException {
 
     }
 
+    private Vector getFriction(GameObject gameObject) {
```

Comment:
this puzzles me. Why do you need separate method? From this method it seems friction coefficient depends from gameObject

inline it